### ISSUE TITLE
Can you add a guide on your wiki or in the documentation for setting up WordPress Coding Standards?

### ISSUE THREAD

**Issue by mikromakro** [Varying-Vagrant-Vagrants/VVV#966]
I'm using Atom Editor on a Mac, and I could really use some details instructions or step-by-step guide to setting up WordPress Coding Standards for VVV. 

[WordPress Coding Standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards)

**Comment by jeremyfelt** [comment]
Agreed. It would be great to have some documentation covering how to use WPCS in various editors and at the command line.

If anyone would like to start writing something up, a pull request against https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org would be excellent.

**Comment by jeffslofish** [comment]
I'm not sure if you were aware of this, but I did some searching and found that WordPress Core has an .editorconfig file in the wordpress directory that basically provides a coding styles spec for all editors to reference.

Many popular editors including Atom have a plugin that supports using this file. 

For more info, you can check out  [http://editorconfig.org/](http://editorconfig.org/).

**Comment by mikromakro** [comment]
jeffslofish, nope i wasn't aware. I will look into it. Thanks.

**Comment by ihorvorotnov** [comment]
@chris10an it's available only in `wordpress-develop`, you can copy it from there. Or get a fresh copy directly from the WordPress Trac [here](https://core.trac.wordpress.org/browser#trunk). And [here](https://core.trac.wordpress.org/ticket/26429) is some background info that might be useful.

**Comment by tomjn** [comment]
WPCS is already setup inside the VM, I don't think it's our place to include documentation for another project inside ours. People should go to the WPCS  project for WPCS documentation
